app/forms/AssistanceDetailsForm.scala

```diff
@@ -26,28 +26,37 @@ object AssistanceDetailsForm {
   val isFastStream = (requestParams: Map[String, String]) =>
     requestParams.getOrElse("applicationRoute", Faststream.toString) == Faststream.toString
 
+  val isEdip = (requestParams: Map[String, String]) =>
+    requestParams.getOrElse("applicationRoute", Edip.toString) == Edip.toString
```

Comment:
Do we definitely want to default to EDIP here?  I thought we were going to fall back to Faststream where the route wasn't available?